.gitignore

```diff
@@ -9,3 +9,4 @@ target/
 /.idea/
 /.idea_modules/
 bin/
+node_modules/
```

Comment:
Actually, shouldn't this be `/node_modules/`?

Reply:
I suppose there wouldn't be `node_modules` anywhere else than at the root, so that makes sense, yes.